pages/about-us/de.text

```diff
@@ -0,0 +1,94 @@
+
+Wir glauben, es ist lebenswichtig, dass essentielle Kommunikationsinfrastruktur durch Organisationen der Bewegungen und nicht Unternehmen oder Regierungen kontrolliert werden sollten.
+
+Wir streben an, Email so sicher und privat wie möglich zu haben. Wir speichern keine IP-Adressen. (Die meisten Dienste speichern detaillierte Verbindungsdaten in ihren Logdateien. Wir speichern nur Informationen, mit denen sich unsere Nutzer*innen nicht identifzieren lassen.). Alle unsere Daten, Email eingenommen, werden verschlüsselt gespeichert. Wir arbeiten hart daran, unsere Server sicher und gut geschützt gegenüber böswilligen Angriffen zu halten. Wir teilen keine Nutzer*innendaten mit irgendjemandem. Wir kämpfen aktiv gegen alle Versuche, uns mittels subpoena oder anderweitig zur Herausgabe von Nutzer*inneninformationen oder Logdateien zu zwingen. Wir werden ein- oder ausgehende Emails nicht lesen, durchsuchen, oder verarbeiten außer um euch mittels automatischer Skripte vor Viren und Spam zu schützen, oder wenn wir durch euch zur Fehlersuche dazu aufgefordert werden.
+
+h2.  Riseups Ziele
+
+!>img/riseup-yellow.gif!
+
+Das Riseup-Kollektiv ist eine autonome Körperschaft in Seattle mit kollektiven Mitgliedern weltweit. Unser Ziel ist es, Hilfe bei der Erschaffung einer freien Gesellschaft zu leisten, eine Welt mit Freiheit von Gier und Meinungsfreiheit, eine Welt ohne Unterdrückung oder Hierarchie, in der Macht gleichmäßig verteilt ist. Unser Beitrag ist das Sicherstellen von Kommunikations- und Computerresourcen für Alliierte in Kämpfen gegen Kapitalismus und andere Unterdrückungsformen.
+
+* Wir schätzen und engagieren uns in Kämpfen für menschliche Befreiung, die ethische Behandlung von Tieren und ökologischer Nachhaltigkeit. Wir beteiligen uns im Kampf für Freiheit und Selbstbestimmung aller von Unterdrückung betroffenen Gruppen. Wir kämpfen gegen jede Form von Vorurteilen, Autoritarismus und Avantgardismus.
+
+* Wir organisieren uns auf der Basis von Autonomie, gegenseitiger Hilfe, Ressourcen- und Wissensaustausch, sozialer Verteidigung, Befreiungsarbeit, Gemeinschaftsbildung und sicherer Kommunikation.
+
+* Wir arbeiten an der Revolution für eine freie Gesellschaft im Hier und Jetzt durch alternative Kommunikationsinfrastruktur, die so gestaltet ist, dass sie das dominante System ersetzen kann.
+
+* Wir bewerben soziale Teilhabe und demokratische Kontrolle über Information, Ideen, Technologie und die Mittel der Kommunikation.
+
+* Wir bestärken Organisationen und Individuen, Technologie bei Befreiungskämpfen zu verwenden. Wir arbeiten an der gegenseitigen Unterstützung bei der Überwindungen der systematischen Unterdrückung, eingebettet in die Verwendung und Entwicklung von Technologie.
```

Comment:
Wir arbeiten an der gegenseitigen Unterstützung beim Überwinden der systematischen Unterdrückung, in deren Rahmen Technologie entwickelt und verwendet wird.